core/driver/sources/src/main/scala/org/virtuslab/ideprobe/reporting/ErrorValidator.scala

```diff
@@ -18,8 +20,16 @@ object ErrorValidator {
     }
   }
 
+  private def ideMessageMatchesMessagesFromConfig(ideMessage: IdeMessage, messagesFromConfig: Seq[String]): Boolean =
+    messagesFromConfig.exists(configMessage =>
```

Comment:
Nit: change to braces for better looks
```suggestion
    messagesFromConfig.exists { configMessage =>
```